### PR TITLE
gui: label tx sizes as "vbytes", and feerates as "sat/vbyte"

### PR DESCRIPTION
**Pull request by SomberNight**
closes https://github.com/spesmilo/electrum/issues/6961

### CODE REVIEW

File: electrum/util.py [hunk 1]
```diff
@@ -792,10 +792,10 @@ def format_satoshis(
...
 FEERATE_PRECISION = 1  # num fractional decimal places for sat/byte fee rates
 _feerate_quanta = Decimal(10) ** (-FEERATE_PRECISION)
-UI_UNIT_NAME_FEERATE_SAT_PER_VBYTE = "sat/byte"
-UI_UNIT_NAME_FEERATE_SAT_PER_VB = "sat/b"
-UI_UNIT_NAME_TXSIZE_VBYTES = "bytes"
-UI_UNIT_NAME_MEMPOOL_MB = "MB"
+UI_UNIT_NAME_FEERATE_SAT_PER_VBYTE = "sat/vbyte"
+UI_UNIT_NAME_FEERATE_SAT_PER_VB = "sat/vB"
+UI_UNIT_NAME_TXSIZE_VBYTES = "vbytes"
+UI_UNIT_NAME_MEMPOOL_MB = "MvB"
```
Comment (SomberNight) [inline electrum/util.py:798]: I am a bit conflicted about this last one ("MvB")...

Comment (SomberNight) [reply electrum/util.py:798]: About block 828682,
- mempool.space says
  ```
  Size	1.54 MB
  Weight	3.99 MWU
  ```
- blockstream.info says
  ```
  Size	1543.694 KB
  Virtual size	999 vKB      <----
  Weight units	3993.305 KWU
  ```

For the mempool diagram,
- johoe's site uses vMB:
  <details>
   <summary>pic</summary>
  
  johoe
  </details>

- mempool.space uses MvB:
  <details>
   <summary>pic</summary>
  
  mempoolspace
  </details>
  
-----

Thoughts?

Comment (accumulator) [reply electrum/util.py:798]: Is it a unit that is virtual, or is it a virtual unit of itself? :)

Comment (accumulator) [reply electrum/util.py:798]: no strong opinion here, but 'virtual megabyte' feels more appropriate than 'mega virtualbyte'

Comment (SomberNight) [reply electrum/util.py]: All right, let's use `vMB` then. I think it looks better, and hopefully less people get confused by it.

**Pull request merged**